### PR TITLE
Adding useful info

### PR DESCRIPTION
**Pull request by jesusgonzalez12**
I've added a way in which users can check their local version of the SDK and a couple of useful links so they can view more docs.

Maybe it would be a good idea to change the title from "Install SDKs" to "Manage SDKs" or to "Getting started with SDKs". Another option is to divide the sections into different files/web pages.

### CODE REVIEW

File: docs/sdks/install-sdks.md [hunk 1]
```diff
@@ -83,4 +104,10 @@ pip install --upgrade pyC8
...
 
 </TabItem>
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+## Reference
+
+You can follow the following links for further reference regarding the SDK's:
+jsC8: [https://www.npmjs.com/package/jsc8](https://www.npmjs.com/package/jsc8)
```
Comment (diana-macrometa) [inline docs/sdks/install-sdks.md]:
```suggestion
- jsC8: [https://www.npmjs.com/package/jsc8](https://www.npmjs.com/package/jsc8)
```

File: docs/sdks/install-sdks.md [hunk 2]
```diff
@@ -83,4 +104,10 @@ pip install --upgrade pyC8
...
+pyC8: [https://pyc8.readthedocs.io/en/latest](https://pyc8.readthedocs.io/en/latest)
```
Comment (diana-macrometa) [inline docs/sdks/install-sdks.md]:
```suggestion
- pyC8: [https://pyc8.readthedocs.io/en/latest](https://pyc8.readthedocs.io/en/latest)
```

File: docs/sdks/install-sdks.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/sdks/install-sdks.md]: Please change the links for both of these to the GitHub repos.

File: docs/sdks/install-sdks.md [hunk 3]
```diff
@@ -64,6 +64,27 @@ To install the SDK in a notebook:
 </TabItem>
 </Tabs>
 
+## View currently installed version of the SDK
```
Comment (diana-macrometa) [inline docs/sdks/install-sdks.md]:
```suggestion
## Check Installed SDK Version
```

File: docs/sdks/install-sdks.md [hunk 4]
```diff
@@ -64,6 +64,27 @@ To install the SDK in a notebook:
...
+
+To know the currently installed version of the SDK, run the following command
```
Comment (diana-macrometa) [inline docs/sdks/install-sdks.md]:
```suggestion
To check which version of the SDK is currently installed, run the following command in your terminal:
```

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
A couple suggestions, then it will be good to go. And you are right, we're getting to the point where we should probably break this topic up into separate ones.

**Comment by jesusgonzalez12** [comment]
Hi @diana-macrometa. Followed recommendations. PR updated. Thanks.

**Comment by diana-macrometa** [comment]
@jesusgonzalez12, the best way to get another review is click the circling arrows next to a reviewer's name. GitHub then sends a notification.

File: docs/sdks/install-sdks.md [hunk 5]
```diff
@@ -83,4 +104,10 @@ pip install --upgrade pyC8
...
 
 </TabItem>
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+## Reference
+
+You can follow the following links for further reference regarding the SDK's:
+jsC8: [https://github.com/macrometacorp/jsC8](https://github.com/macrometacorp/jsC8)
```
Comment (diana-macrometa) [inline docs/sdks/install-sdks.md]:
```suggestion
- jsC8: [https://github.com/macrometacorp/jsC8](https://github.com/macrometacorp/jsC8)
```

File: docs/sdks/install-sdks.md [hunk 6]
```diff
@@ -83,4 +104,10 @@ pip install --upgrade pyC8
...
+pyC8: [https://github.com/Macrometacorp/pyC8](https://github.com/Macrometacorp/pyC8)
```
Comment (diana-macrometa) [inline docs/sdks/install-sdks.md]:
```suggestion
- pyC8: [https://github.com/Macrometacorp/pyC8](https://github.com/Macrometacorp/pyC8)
```

**Review by diana-macrometa — APPROVED** [review]
Looks good!

**Comment by jesusgonzalez12** [comment]
Thanks for the tip @diana-macrometa, will take it into account next time

**Pull request merged**